src/components/ThemeProvider/tests/ThemeProvider.test.tsx

```diff
@@ -120,4 +120,19 @@ describe('<ThemeProvider />', () => {
       }),
     );
   });
+
+  it('sets color system properties when global theming is enabled', () => {
+    const themeProvider = mountWithAppProvider(
+      <ThemeProvider theme={{}}>
+        <p>Hello</p>
+      </ThemeProvider>,
+      {features: {unstableGlobalTheming: true}},
+    );
+
+    expect(themeProvider.find('div').props().style).toStrictEqual(
```

Comment:
I was trying to do an assertion on body style for background color and color here but for some reason it is returning an empty string. I tried forcing an update in case it was a render cycle issue, but no luck.

Reply:
This is a big part of the reason we avoided setting these on the body in the first place. Take a look at our `setRootProperty` function to see that it is totally untested. The Jest env seems not to give us the ability to realistically assert on the `body`.